pages/_app.js

```diff
@@ -1,17 +1,20 @@
 import Navigation from "@/components/Navigation";
 import GlobalStyle from "@/styles";
 import Head from "next/head";
-import { createContext } from "react";
+import { createContext, useState } from "react";
 import { v4 as uuidv4 } from "uuid";
 import useLocalStorageState from "use-local-storage-state";
 
 export const myProjectsContext = createContext();
+export const myViewsContext = createContext();
```

Comment:
do you really need contexts? 

this seems to be overkill here, since you could also pass props.

Reply:
I thought if we use dynamic routes its more clever to handle the global state with createContext? I changed it but for me its more comfortable to use createContext